### PR TITLE
Sara Kuddah

### PR DESCRIPTION
**Pull request by Sara-Kuddah**

### CODE REVIEW

File: exercise/js/main.js [hunk 1]
```diff
@@ -54,46 +58,84 @@ $("#button1").on('click', question1);
...
 
+};
+
+$("#button3").click(question9);
 /**
  * Question 10:
  * Try to do Question 9 but with only one jQuery call. $("selector").css({attribute: "value", attribute: "value"})
  */
 
+function question9() {
```
Comment (Sager-Alarifi-GA) [inline exercise/js/main.js]: should be named question 10

File: exercise/js/main.js [hunk 2]
```diff
@@ -54,46 +58,84 @@ $("#button1").on('click', question1);
...
+
+$("#button3").click(question8);
 
 /**
  * Question 9:
  * When button 3 is clicked, select all paragraphs on the page and change the color to blue and font to Georgia.
  */
 
+function question9() {
+	$(".wrapper").css("color", "blue");
```
Comment (Sager-Alarifi-GA) [inline exercise/js/main.js]: all paragraphs should be $("p") not $(".wrapper")

**Comment by Sager-Alarifi-GA** [comment]
Good job, almost complete, please fix the issues and push.
Marked incomplete.

File: exercise/js/main.js (hunk 2 above)
Comment (Sara-Kuddah) [reply exercise/js/main.js]: done :)

File: exercise/js/main.js (hunk 1 above)
Comment (Sara-Kuddah) [reply exercise/js/main.js]: done :)

**Comment by Sager-Alarifi-GA** [comment]
Good job,
Marked as complete.

**Pull request closed without merging**